docs/source/asr/intro.rst

```diff
@@ -1,34 +1,156 @@
+
+.. code-block:: bash
+
+    python eval_beamsearch_ngram.py nemo_model_file=<path to the .nemo file of the model> \
+        input_manifest=<path to the evaluation JSON manifest file \
+        kenlm_model_file=<path to the binary KenLM model> \
+        beam_width=[<list of the beam widths, separated with commas>] \
+        beam_alpha=[<list of the beam alphas, separated with commas>] \
+        beam_beta=[<list of the beam betas, separated with commas>] \
+        preds_output_folder=<optional folder to store the predictions> \
+        probs_cache_file=null \
+        decoding_mode=beamsearch_ngram \
+        decoding_strategy="<Beam library such as beam, pyctcdecode or flashlight>"
+
+See more information about LM decoding :doc:`here <./asr_language_modeling>`.
+
+Use real-time transcription
+---------------------------
+
+It is possible to use NeMo to transcribe speech in real-time. You can find an example of how to do 
+this in the following `notebook tutorial <https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/Online_ASR_Microphone_Demo.ipynb>`_.
+
+
+Try different ASR models
+------------------------
+
+NeMo offers a variety of open-sourced pretrained ASR models that vary by model architecture:
+
+* **encoder architecture** (FastConformer, Conformer, Citrinet, etc.),
+* **decoder architecture** (Transducer, CTC & hybrid of the two),
+* **size** of the model (small, medium, large, etc.).
+
+The pretrained models also vary by:
+
+* **language** (English, Spanish, etc., including some **multilingual** and **code-switching** models),
+* whether the output text contains **punctuation & capitalization** or not.
+
+The NeMo ASR checkpoints can be found on `HuggingFace <https://huggingface.co/models?library=nemo&sort=downloads&search=nvidia>`_, or on `NGC <https://catalog.ngc.nvidia.com/models?query=nemo&orderBy=weightPopularDESC>`_. All models released by the NeMo team can be found on NGC, and some of those are also available on HuggingFace. 
+
+All NeMo ASR checkpoints open-sourced by the NeMo team follow the following naming convention: 
+``stt_{language}_{encoder name}_{decoder name}_{model size}{_optional descriptor}``.
+
+You can load the checkpoints automatically using the ``ASRModel.from_pretrained()`` class method, for example:
+
+.. code-block:: python
+
+    import nemo.collections.asr as nemo_asr
+    # model will be fetched from NGC
+    asr_model = nemo_asr.models.ASRModel.from_pretrained("stt_en_fastconformer_transducer_large")
+    # if model name is prepended with "nvidia/", the model will be fetched from huggingface
+    asr_model = nemo_asr.models.ASRModel.from_pretrained("nvidia/stt_en_fastconformer_transducer_large")
+    # you can also load open-sourced NeMo models released by other HF users using:
+    #asr_model = nemo_asr.models.ASRModel.from_pretrained("<HF username>/<model name>")
```

Comment:
Maybe uncomment this line?

Reply:
I would rather leave it commented, since it won't work if it is run as-is